README.md

```diff
@@ -33,8 +34,63 @@ aimed at react developers that want to embed the widgets into their applications
 npm install --save webex @webex/widgets
 ```
 
+## Webex Components vs Webex Widgets
+
+[Webex Widgets](https://github.com/webex/widgets#webex-widgets) are based on [Webex components](https://github.com/webex/components#webex-components) but include the adapter that talks to our [Javascript SDK](https://github.com/webex/webex-js-sdk).
+This means that the Webex Widgets use the [SDK Component Adapter](https://github.com/webex/sdk-component-adapter#webex-sdk-component-adapter) to inject the [Webex data](https://github.com/webex/webex-js-sdk#webex-js-sdk) for you.
+You also have to take the Widget layout as-is, but the benefit is that there are no configurations needed.
```

Comment:
Is there a reason why you didn't include the introduction sentence of the Webex Component System?
This sentence now it's odd and without context, because it says "also", and when I read it it's like, "also from what?", "what are you comparing this to?"

I would appreciate you include the intro sentence or something similar

Reply:
If you don't want to talk about the component features, I would remove "also" because it still sounds comparative